pkg/pillar/utils/proc.go

```diff
@@ -101,7 +106,7 @@ func GetPidFromFileTimeout(pidFile string, timeoutSeconds uint, wk *WatchdogKick
 			wk.ps.StillRunning(wk.agentName, wk.warnTime, wk.errTime)
 		}
 
-		time.Sleep(500 * time.Millisecond)
+		time.Sleep(100 * time.Millisecond)
```

Comment:
The warntime and errtime are orders of magnitude larger than this. Have you tun into cases when warntime expired when you where running this every 500ms?